ada/language/ast.py

```diff
@@ -3586,6 +3586,14 @@ def is_view_of_type(comp_view=T.BaseTypeDecl.entity):
     def is_array_or_rec():
         return Entity.is_array | Entity.is_record_type
 
+    @langkit_property(public=True, return_type=Bool)
+    def is_inherited(p=T.BasicDecl.entity):
```

Comment:
I'd rather you be even more descriptive in the property name: `is_inherited_primitive`. That will make it clearer without the programmer having to check whether the node this is called on is a type or a field or a primitive or whatever.

Reply:
Agreed, I changed it.